### ISSUE TITLE
Module now has a limit to images per page

### ISSUE THREAD

**Issue by ChemiKyle** [ctsit/imagemap#58]
This affects both Chrome and Firefox, and REDCap 10.5.1 and 10.0.1 on my machine as well as @mbentz-uf. This also occurs on a project I just made on `stage_b` (PID 21) which is running 10.3.3.

The following error appears in the console:

```
Uncaught Can't access data until binding complete. bundle.js:234:27834
```

It appears to be rooted in a timeout.

Strangely, the first 2 image maps from `Instrument Example.zip` appear, the second does not alter radio buttons, and the rest do not appear at all.

This error does not appear when rolling back to commit 9c8ae5a (though there are many console errors when selecting things) however, commit 6e9499e (which is the head of my PR #54) once again fails.  
Something has changed since October 2020 beyond of the scope of this module itself.

**Comment by 123andy** [comment]
I'm asking Jae from my team to look into this as well.  This is a priority to fix for a researcher in my team and need to get this out -- I'm okay removing other changes for now -- just the ability to do API endpoints.

**Comment by ChemiKyle** [comment]
Issue may be rectified by providing some additional args to the initialization of the imageMap object itself https://github.com/jamietre/ImageMapster/issues/96

**Comment by ChemiKyle** [comment]
There appears to be a ~ 2 image limit now. `Instrument Example.zip` has too many images for now.

**Comment by 123andy** [comment]
I found a work-around for this.  I'm making a new pull request.

**Comment by mbentz-uf** [comment]
Addressed by PR #61.
